### ISSUE TITLE
Model references generated incorrectly when providing `schema`

### ISSUE THREAD

**Issue by jsau-** [sequelize/sequelize-auto#501]
Hi! Had some issues generating models when providing the schema flag (both programmatically and using CLI).

The crux of the issue seems to be that when providing a schema, model references are generated like:

```
references: {
  model: {
    tableName: 'groups',
    schema: 'public'
  },
  key: 'id'
}
```

When not providing a schema, model references are generated like:

```
references: {
  model: 'groups',
  key: 'id'
}
```

When providing a schema flag, this results in a type error, eg.

```
GroupUser.ts:41:15 - error TS2322: Type 'string' is not assignable to type '<M extends Model<any, any>>(this: ModelStatic<M>, schema: string, options?: SchemaOptions | undefined) => (new () => M) & typeof Model'.

41               schema: 'public',
```

I've made a small-scale reproduction at https://github.com/jsau-/sequelize-auto-schema-incorrect-references-repro which seems to reliably re-create the issue for me. The `README` inside should include some basic information about how to run it. 

Dependencies:
* [Installed locally] Postgres 12
* [Installed locally] Node 12.X
* Typescript 4.1.5
* sequelize 6.5.0
* sequelize-auto 0.7.11
* pg 8.5.1
* pg-hstore 2.3.3

Other things I've tried:
* Changing Postgres version back to a much older version outright results in invalid models being generated
* Passing a `sequelize` instance when instantiating `sequelize-auto` doesn't change anything

I'll try spend some time this afternoon digging into the project source, see if anything jumps out.

Edits:

* The relevant project file seems to be [auto-generators.ts::260](https://github.com/sequelize/sequelize-auto/blob/ea6361e426efdf8c13fc9559f56d68546b8151e9/src/auto-generator.ts#L260). Removing the `if` block and just writing out as if schema weren't specified does get us past the initial compilation error. Digging into the implications of omitting that now to see what impact that has.
* Having removed that block from my local `node_modules`, and re-created models this all seems to be working (including when there are multiple tables named exactly the same across multiple schemas). Is there anything anyone else is aware of I should check?

**Comment by steveschmitt** [comment]
It seems like the `references` should be able to provide the schema name _somehow_, but obviously we aren't doing it correctly.

**Comment by steveschmitt** [comment]
Thanks for your work on this.  I think I will remove the "if" block for now, and just use the form:
```js
references: {
  model: 'groups',
  key: 'id'
}
```
for all cases.  

If it turns out not to work for some combination of schemas, I will look into it again.

**Comment by steveschmitt** [comment]
Fixed in 0.8.0
